core/src/test/java/com/klaytn/caver/common/KeyringContainerTest.java

```diff
@@ -108,118 +111,113 @@ public void checkAddress(List<String> expectedAddress, KeyringContainer containe
-            List<String> expectAddressList = container.generate(10, "entropy");
+            List<String> expectAddressList = caver.wallet.generate(10, "entropy");
 
-            checkAddress(expectAddressList, container);
+            checkAddress(expectAddressList);
         }
     }
 
     public static class newKeyringTest {
+        Caver caver = new Caver(Caver.DEFAULT_URL);
 
         //CA-KEYRINGCONTAINER-003
         @Test
         public void newSingleKeyring() {
-            KeyringContainer container = new KeyringContainer();
-            PrivateKey privateKey = PrivateKey.generate();
+            PrivateKey privateKey = caver.wallet.keyring.generate().getKey();
             String expectAddress = privateKey.getDerivedAddress();
             String expectPrivateKey = privateKey.getPrivateKey();
 
-            AbstractKeyring added = container.newKeyring(expectAddress, expectPrivateKey);
+            AbstractKeyring added = caver.wallet.newKeyring(expectAddress, expectPrivateKey);
             validateSingleKeyring(added, expectAddress, expectPrivateKey);
-            assertEquals(1, container.length());
+            assertEquals(1, caver.wallet.length());
         }
 
         //CA-KEYRINGCONTAINER-004
         @Test
         public void newMultipleKeyring() {
-            KeyringContainer container = new KeyringContainer();
-            String expectAddress = PrivateKey.generate().getDerivedAddress();
+            String expectAddress = caver.wallet.keyring.generate().getAddress();
             String[] expectPrivateKeyArr = {
-                    PrivateKey.generate().getPrivateKey(),
-                    PrivateKey.generate().getPrivateKey(),
-                    PrivateKey.generate().getPrivateKey(),
+                    caver.wallet.keyring.generate().getKey().getPrivateKey(),
+                    caver.wallet.keyring.generate().getKey().getPrivateKey(),
+                    caver.wallet.keyring.generate().getKey().getPrivateKey(),
             };
 
-            AbstractKeyring added = container.newKeyring(expectAddress, expectPrivateKeyArr);
+            AbstractKeyring added = caver.wallet.newKeyring(expectAddress, expectPrivateKeyArr);
             validateMultipleKeyring(added, expectAddress, expectPrivateKeyArr);
         }
 
         //CA-KEYRINGCONTAINER-005
         @Test
         public void newRoleBasedKeyring() {
-            KeyringContainer container = new KeyringContainer();
-            String expectAddress = PrivateKey.generate().getDerivedAddress();
+            String expectAddress = caver.wallet.keyring.generate().getAddress();
             String[][] expectPrivateKeyArr = {
                     {
-                        PrivateKey.generate().getPrivateKey(),
-                        PrivateKey.generate().getPrivateKey(),
-                        PrivateKey.generate().getPrivateKey(),
+                            caver.wallet.keyring.generate().getKey().getPrivateKey(),
```

Comment:
Why you use a caver.wallet.keyring.generate()?
caver.wallet.keyring.generate() makes a SingleKeyrinng instance.

Reply:
I think more proper way to handle this is to use `caver.wallet.keyring.generateSingleKey()`.

In [caver-js/test/packages/caver.wallet#L184](https://github.com/klaytn/caver-js/blob/ae1b51295614131c364b1e3e16a82a98bb629e2e/test/packages/caver.wallet.js#L184), it creates a private key string by calling `caver.wallet.keyring.generateSingleKey()` which is also implemented in caver-java.

@sirano11 
If you are agree with this, I'll change `caver.wallet.keyring.generate()` to `caver.wallet.keyring.generateSingleKey()`.

Thanks for your feedback.